docs/meshstack.meshmarketplace.tenant-services.md

```diff
@@ -3,15 +3,15 @@ id: meshstack.meshmarketplace.tenant-services
 title: Tenant Services
 ---
 
+> **The Tenant Services feature is deprecated! Please do not use this anymore.**
```

Comment:
If its deprecated already why dont we remove the text below?

Reply:
Doesn't deprecated mean planned for deletion? If not why not remove all the other stuff too?